Formula/grpcui.rb

```diff
@@ -0,0 +1,20 @@
+require "English"
```

Comment:
What's this for?

Reply:
This was addressing a style issue reported by CI: https://github.com/Homebrew/homebrew-core/runs/799593601

Reply:
@dawidd6, I see you fixed this and approved the PR. Thanks! Will it be merged soon? Or is there any other action I need to take before getting this in?

Reply:
Will be merged soon.